src/screens/onboarding/components/step-content/experience-content.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useCallback } from 'react';
+import { NativeStackNavigationProp } from '@react-navigation/native-stack';
+import { useNavigation } from '@react-navigation/native';
+import { ScrollView, View } from 'react-native';
+
+import { HeadingLevel, OnboardingRoute } from 'src/common/enums';
+import { OnboardingStackParamList } from 'src/common/types';
+import { Heading } from 'src/components';
+import { AddButton } from '../';
+import styles from './styles';
+
+type ExperienceContentScreenProps = NativeStackNavigationProp<
```

Comment:
Do this for other components too
```suggestion
type ExperienceContentProps = NativeStackNavigationProp<
```